vagrant/config/boxes.all.yaml

```diff
@@ -0,0 +1,22 @@
+operating_systems:
+  - centos7
+  - debian9
+  - debian10
+  - ubuntu1604
+  - ubuntu1804
+  - fedora26
+  - fedora27
+  - fedora28
+  - fedora29
+  - fedora30
+
+foreman_versions:
+  - '1.20'
+  - '1.21'
+  - '1.22'
+  - '1.23'
+  - '1.24'
+  - 'nightly'
```

Comment:
nit: these don't need to be quoted

Reply:
True, but 1.20, 1.30 etc needs to be, and I must admit I like consistency :)

Reply:
They shouldn't need to be quoted either afaik

Reply:
1.20 is parsed as a float and becomes 1.2 :(

Reply:
Ouch, yeah that's a gotcha :)